### DISCUSSION TITLE
Blue Skies

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#927] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/blue-skies
+ 2 dimensions
+ A few biomes
+ Tens of trees

**Comment by YharonStone** [comment]
This mod is realy impressive, it iven has its own sond track! 

and most of the trees in the mod are quite simple

**Comment by Mythril1212** [comment]
Please make this compat happen, its been a long time
